style.css

```diff
@@ -473,7 +514,66 @@ button:hover {
   background-repeat: no-repeat;
   background-size: contain;
   padding: 10px;
-  margin-left: -10px;
+}
+
```

Comment:
- [ ] Great work @Tracy-miranja, you have well created the form and the submission works fine too, but I think we have a few broken parts in the design, see the record below for more details, I believe you can see the white space on the right of your page, right? please fix that.
- [ ] Please invite your collaborator to contribute to this milestone as it is mentioned in the project requirement.

<img width="956" alt="1" src="https://user-images.githubusercontent.com/77843050/207342117-d67cf475-bec7-4ce7-966b-6b839ff5278e.png">


![5](https://user-images.githubusercontent.com/77843050/207329566-cc5f950f-dfb3-41bb-b5a3-2a2f7b0a48b1.gif)